src/code-server/install.sh

```diff
@@ -17,12 +17,18 @@ do
     code-server --install-extension "$extension"
 done
 
+CODE_SERVER_WORKSPACE="$_REMOTE_USER_HOME"
+
+if [ -n "$WORKSPACE" ]; then
```

Comment:
```suggestion
if [[ -n $WORKSPACE ]]; then
```
If we're using Bash, let's commit and simplify (in this case `[[` allows omitting quoting).